pkg/daemon/server/service/rater/rater_test.go

```diff
@@ -107,8 +106,8 @@ func TestRater_Start(t *testing.T) {
 	}
 	r := NewRater(ctx, pipeline)
 	podTracker := NewPodTracker(ctx, pipeline, WithRefreshInterval(time.Second*5))
-	podTracker.httpClient = &mockHttpClient{podOneCount: 0, podTwoCount: 0, lock: &sync.RWMutex{}}
-	r.httpClient = &mockHttpClient{podOneCount: 0, podTwoCount: 0, lock: &sync.RWMutex{}}
+	//podTracker.httpClient = &mockHttpClient{podOneCount: 0, podTwoCount: 0, lock: &sync.RWMutex{}}
+	//r.httpClient = &mockHttpClient{podOneCount: 0, podTwoCount: 0, lock: &sync.RWMutex{}}
```

Comment:
why do you have to comment these? isn't `t.SkipNow()` enough?

Reply:
lint was failing